src/protagonist/API.Client/DlcsClient.cs

```diff
@@ -191,6 +191,36 @@ public async Task<bool> DeletePortalUser(string portalUserId)
         return await response.ReadAsHydraResponseAsync<Image>(jsonSerializerSettings);
     }
 
+    public async Task<Image?> ReingestImage(int spaceId, string imageId)
+    {
+        var uri = $"customers/{currentUser.GetCustomerId()}/spaces/{spaceId}/images/{imageId}/reingest";
+        var response = await httpClient.PostAsync(uri, null);
+        return await response.ReadAsHydraResponseAsync<Image>(jsonSerializerSettings);
+    }
+
+    public async Task<bool> DeleteImage(int spaceId, string imageId)
+    {
+        var uri = $"customers/{currentUser.GetCustomerId()}/spaces/{spaceId}/images/{imageId}";
+        try
+        {
+            var response = await httpClient.DeleteAsync(uri);
+            return true;
+        }
+        catch (Exception ex)
+        {
+            logger.LogError(ex, "Error deleting image '{imageId}'", imageId);

```

Comment:
Tiny thing but structured logging properties are all pascal case elsewhere in Protagonist but this uses camel case